docs/source/case_study_a5.rst

```diff
@@ -26,17 +43,36 @@ on the `Skellefte river <https://en.wikipedia.org/wiki/Skellefte_River>`_,
 which models one week of operation of the fifteen power stations
 along the river.
 
-This tutorial provides a step-by-step guide to run Case Study A5 on Spine Toolbox
-and is organized as follows:
+In this tutorial we first introduce some key aspects of the model and then provide 
+a step-by-step guide on how to build a workflow, which includes building a model.
+Finally, we execute the workflow and examine the results. 
+The tutorial is organized as follows:
 
 .. contents::
    :local:
+   
+   
+Spine Installation
+-------------------
 
+.. note:: This tutorial is written for `Spine Toolbox 
+   <https://github.com/Spine-project/Spine-Toolbox/>`_ and `SpineOpt 
+   <https://github.com/Spine-project/SpineOpt.jl>`_ development version on |Revised|.
+   If you are using a later version, the steps in this setup guide may be outdated.   
 
-Introduction
-------------
+#. Install Spine Toolbox as described in `installing-spine-toolbox 
+   <https://github.com/Spine-project/Spine-Toolbox#installing-spine-toolbox>`_.
+
+#. Download and install Julia 1.5 (or greater) as described `here <https://julialang.org/downloads/>`_.
+
+#. Install SpineOpt: Launch Spine Toolbox and select
+   **File -> Tool configuration assistants... -> SpineOpt.jl** from the main menu.   
```

Comment:
I'm not sure what to do here because the File->Tool Configuration Assistants...->SpineOpt.jl is gone. We now have a 'Add/Update SpineOpt Wizard' that does the same job but it's in Settings->Tools.

Reply:
Maybe it's enough to refer to the **Add/Update SpineOpt Wizard** button in Settings->Tools instead of the config assistant.